app/migrations/Version20221128145933.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+use Mautic\PluginBundle\Helper\IntegrationHelper;
+use Mautic\SmsBundle\Form\Type\ConfigType;
+
+final class Version20221128145933 extends AbstractMauticMigration
+{
+    /**
+     * @throws SkipMigration
+     */
+    public function preUp(Schema $schema): void
+    {
+        /** @var IntegrationHelper $integrationHelper */
+        $integrationHelper = $this->container->get('mautic.helper.integration');
+        $integration       = $integrationHelper->getIntegrationObject('Twilio');
+        $settings          = $integration->getIntegrationSettings()->getFeatureSettings();
+        if (empty($settings['disable_trackable_urls'])) {
+            throw new SkipMigration('Schema includes this migration');
+        }
+    }
+
+    public function up(Schema $schema): void
+    {
+        $confFile = dirname(__DIR__).'/config/local.php';
+
+        if (!file_exists($confFile)) {
+            return;
+        }
+
+        require $confFile;
+
+        $parameters[ConfigType::SMS_DISABLE_TRACKABLE_URLS] = 1;
```

Comment:
Shouldn't the value from `$settings['disable_trackable_urls']` be used here? This is disabling it for everyone no matter if they have it already enabled or disabled.

Reply:
Migration continue just If was option enabled in plugin settings https://github.com/mautic/mautic/pull/11683/files#diff-32cb6e34e8def8e50b5fed6856fa25e60ef9d212f3875c094cddde67989e6917R25
Other ways keep default value.

Reply:
Gotcha. All good then.